minecraft-server/start-minecraft.sh

```diff
@@ -252,6 +289,11 @@ do
     cp -rf "$c" /data/config
   fi
 done
-
+if [ "$TYPE" = "SPIGOT" ]; then
```

Comment:
Again, this catches BUKKIT types as well.  We use a separate volume for plugins as they are not compatible with mods though they look like them